pubsub/cloud-client/publisher_test.py

```diff
@@ -45,6 +46,20 @@ def topic(client):
     yield response.name
```

Comment:
How about adding a TODO here to delete the topic afterwards. It would be nice to have an explanation of why it's not possible at the moment.

Reply:
Done.